backend/src/main/java/gov/cdc/usds/simplereport/service/PatientLinkService.java

```diff
@@ -37,23 +33,16 @@ public PatientLink getPatientLink(String internalId) {
             () -> new IllegalGraphqlArgumentException("No patient link with that ID was found"));
   }
 
-  public Organization getPatientLinkCurrent(String internalId) {
-    PatientLink pl = getPatientLink(internalId);
-
-    if (pl.getRefreshedAt().after(Date.from(Instant.now().minus(oneDay, ChronoUnit.HOURS)))) {
-      return pl.getTestOrder().getOrganization();
-    } else {
-      throw new InvalidPatientLinkException(
-          "Patient Link is expired; please contact your provider");
-    }
-  }
-
-  public boolean verifyPatientLink(String internalId, LocalDate birthDate) {
+  public boolean verifyPatientLink(String internalId, LocalDate birthDate)
```

Comment:
While we're here can we move the UUID conversion out of the service layer and into either the controller or the servlet?

Reply:
Yep, done!